src/bin/inx-chronicle/main.rs

```diff
@@ -44,6 +44,27 @@ async fn main() -> eyre::Result<()> {
         ByteSize::b(db.size().await?)
     );
 
+    match db
+        .collection::<ApplicationStateCollection>()
+        .get_last_migration()
+        .await?
+        .as_deref()
+    {
+        None => {
+            info!("Setting migration version to {}", migrations::LATEST_VERSION);
+            db.collection::<ApplicationStateCollection>()
+                .set_last_migration(migrations::LATEST_VERSION)
+                .await?;
+        }
+        Some(migrations::LATEST_VERSION) => (),
+        Some(v) => {
+            eyre::bail!(
+                "Invalid database migration version {}, please run the `migrate` command.",
```

Comment:
Small suggestion:
```rust
eyre::bail!("Expected database migration version {}, but found {}. Please run the `migrate` command.", migrations::LATEST_VERSION, v);
```

Reply:
Well, there is no expected version, just a list of migrations and the versions they are from. And I dunno, I get that the command may change but I think it's better for errors to be explicit when telling you what to do.

Reply:
Aren't we expecting the db to have been migrated to the LATEST_VERSION?

Reply:
Let's get this merged, we can always improve the logging.